### DISCUSSION TITLE
[Fundamentos de C#] [Mão na massa: Editor de Textos]

### DISCUSSION THREAD

**Discussion by FehCastro** [balta-io/forum#438] [Tira Dúvidas] [answered]
![image](https://github.com/balta-io/forum/assets/108245423/6257badd-f479-4596-bd00-8507c69143a4)

Olá a todos estou tendo um problema que está aparecendo nessa foto e não sei como arrumar.

Estou utilizando essa configuração do .Net 8.0:

E quando eu executo o dotnet run aparece a seguinte mensagem:

C:\Dev\TextEditor\Program.cs(45,21): error CS0200: A propriedade ou o indexador "ConsoleKeyInfo.Key" não pode ser atribuído, pois é somente 
leitura [C:\Dev\TextEditor\TextEditor.csproj]

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @FehCastro, está faltando um `=`.. são dois iguais (`==`).
= -> atribuição
== -> Comparação

**Reply by FehCastro** [reply]
Olá, Balta realmente com o sinal de comparação [ == ] o erro saiu, porém, no vídeo era usado o símbolo de comparação diferente e ai eu descobri que estava usando o símbolo errado [ |= ]  e o correto é [ != ] . Valeu.
